### PR TITLE
Main dialog use system font size

### PR DESCRIPTION
**Pull request by FlexW**

### CODE REVIEW

File: src/gui/UserStatusSelector.qml [hunk 1]
```diff
@@ -24,10 +24,6 @@ ColumnLayout {
     spacing: 0
     property NC.UserStatusSelectorModel userStatusSelectorModel
 
-    FontMetrics {
-        id: metrics
-    }
-
     Text {
```
[the remark this answers is not in the document]
Comment (FlexW) [reply src/gui/UserStatusSelector.qml]: Okay, done

**Review by claucambra — APPROVED** [review]

**Comment by allexzander** [comment]
@mgallien Are you fin with this one?

**Review by allexzander — APPROVED** [review]

**Comment by claucambra** [comment]
\rebase

**Review by mgallien — APPROVED** [review]

**Pull request merged**